packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -226,6 +226,9 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
           _checkDragOffset(notification.metrics.viewportDimension);
         }
       }
+      if (_mode == _RefreshIndicatorMode.armed && notification.dragDetails == null) {
```

Comment:
It's not clear (to me anyway) that this particular line is intended to "Start refresh while bouncing back to 0.0 from overscroll on iOS". Why is this iOS specific and etc. Either I need to read a bunch more of the code, or some kind of comment should appear here.

Reply:
Added comment.